build/golint.sh

```diff
@@ -24,6 +24,13 @@ echo "Running golangci-lint..."
 
 golangci-lint run --timeout ${TIMEOUT} --skip-dirs ${SKIP_DIR_REGEX} -E maligned,whitespace,gocognit,unparam -e '`ctx` is unused'
 
+# gofmt should run everywhere, including
+#   1. Skipped directories in previous step
+#   2. Build exempted files using build tags
+#      Note: Future build tags should be included.
+echo "Running gofmt..."
+golangci-lint run --timeout ${TIMEOUT} --disable-all --enable gofmt --build-tags integration
```

Comment:
@Bhargav-InfraCloud If we add this, we don't need the previous call (Line no 25), right?

Reply:
It would **require both**, actually. The first one (line 25) runs 7 default linters (`errcheck`, `gosimple`, `govet`, `ineffassign`, `staticcheck`, `typecheck` & `unused`) with 4 enabled (`maligned`, `whitespace`, `gocognit` & `unparam`).

The second (line 32) only runs `gofmt` which is the core of this change.

Ideally, `gofmt` could've been added to the first. But in doing so, the extra conditions (skipping the directory and excluding ctx error) will get applied to this which is not wise. And also, `gofmt` without `--build-tags integration` would ignore 2 files dropping the % in report. Enable it in first directly and it would throw a lot of unexpected linter errors (not required as of now).

As I suggested in PR, using `.golangci-yaml` can solve this issue, where we can specify all the configs and run it only once. But that would be for future PR if agreed upon.

Reply:
Thanks @Bhargav-InfraCloud could you please file an issue to switch to `.golanci-yaml`

Reply:
Opened #1761